src/components/BrowseRecipes.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+
+import { IRecipe } from '../models/IRecipe';
+
+export interface BrowseRecipesProps {
+  recipes: IRecipe[];
+}
+
+export const BrowseRecipes: FC<BrowseRecipesProps> = ({ recipes }) => {
+  console.log(recipes)
+
+  const recipeTitles = [...recipes.map(recipe => recipe.title)]
+
+  return (
+    <>
+      <div>
+        <label htmlFor="title-values">Recipes: </label>
+        {/* TODO: make these links of some sort */}
+        <span id="title-values">{recipeTitles}</span>
```

Comment:
You probably want to do something like `recipeTitles.join(', ')`

Reply:
I am going to make these some kind of clickable link or something, I just put this as a placeholder for now.